core/src/main/java/org/apache/iceberg/rest/RESTSerializers.java

```diff
@@ -115,7 +117,9 @@ public static void registerAll(ObjectMapper mapper) {
         .addSerializer(LoadViewResponse.class, new LoadViewResponseSerializer<>())
         .addSerializer(ImmutableLoadViewResponse.class, new LoadViewResponseSerializer<>())
         .addDeserializer(LoadViewResponse.class, new LoadViewResponseDeserializer<>())
-        .addDeserializer(ImmutableLoadViewResponse.class, new LoadViewResponseDeserializer<>());
+        .addDeserializer(ImmutableLoadViewResponse.class, new LoadViewResponseDeserializer<>())
+        .addSerializer(GetScanTasksResponse.class, new GetScanTasksResponseSerializer<>())
+        .addDeserializer(GetScanTasksResponse.class, new GetScanTasksResponseDeserializer<>());
```

Comment:
what about CreateScanRequest deserializer and deserializer?

Reply:
I believe we actually do not need to define custom serializers of our own for the `CreateScanRequest`. For example this class does not have any serializers for `CreateTableRequest`. 

I think the reason I had to add serializers specifically for `GetScanTasks` was that I need to add this custom parsing with the `GetScanTasksResponseParser`.